pkg/cmd/roachtest/tests/ts_util.go

```diff
@@ -38,9 +38,13 @@ const (
 	rate
 )
 
+// defaultSampleNanos is the default sampling period for getMetrics.
+const defaultSampleNanos = int64(time.Minute)
```

Comment:
I think it would be more readable to have it as
```
const defaultSamplePeriod = time.Minute
```
and then use `defaultSamplePeriod.Nanoseconds()` only when using it in `TimeSeriesQueryRequest`.